services/NetflixService.js

```diff
@@ -30,9 +30,12 @@ module.exports.getGenres = async (req, res) => {
         
     // Extract data from the response
     const apiData = response.data.results;
-
+    const responseData = {
```

Comment:
In genres we dont have any url in response i think we can avoid this url to append in response.

Reply:
Sure, I will fix it.